packages/quantic/force-app/main/default/lwc/quanticDateFacet/__tests__/quanticDateFacet.test.js

```diff
@@ -210,9 +299,257 @@ describe('c-quantic-date-facet', () => {
+        ...initialFacetState,
+        isLoading: true,
+        hasError: false,
+        firstSearchExecuted: false,
+      };
+    });
+
+    it('should display the placeholder component', async () => {
+      const element = createTestComponent();
+      await flushPromises();
 
-    afterAll(() => {
-      functionsMocks.buildDateFacet.mockReturnValue(dateFacetControllerMock);
+      const placeholder = element.shadowRoot.querySelector(
+        selectors.placeholder
+      );
+      expect(placeholder).not.toBeNull();
+    });
+
+    describe('when the interface has an error', () => {
+      beforeEach(() => {
+        searchStatusState = {
+          ...initialFacetState,
+          isLoading: true,
+          hasError: true,
+          firstSearchExecuted: false,
+        };
+      });
+
+      it('should not display the placeholder component', async () => {
+        const element = createTestComponent();
+        await flushPromises();
+
+        const placeholder = element.shadowRoot.querySelector(
+          selectors.placeholder
+        );
+        expect(placeholder).toBeNull();
+      });
+    });
+  });
+
+  describe('when the component is initialized', () => {
+    describe('when the facet has values', () => {
+      beforeEach(() => {
+        facetState = {
+          ...initialFacetState,
+          values: exampleFacetValues,
+        };
+      });
+
+      it('should display the facet card when the facet values have results', async () => {
+        const element = createTestComponent();
+        await flushPromises();
+
+        const cardContainer = element.shadowRoot.querySelector(
+          selectors.cardContainer
+        );
+        expect(cardContainer).not.toBeNull();
+      });
+
+      it('should display the facet values', async () => {
+        const expectedFacetValues = exampleFacetValues.map((facetValue) => ({
+          ...facetValue,
+          checked: false,
+          highlightedResult: facetValue.value,
+        }));
+        const element = createTestComponent({
+          ...defaultOptions,
+          noSearch: true,
+        });
+        await flushPromises();
+
+        const facetValueElements = element.shadowRoot.querySelectorAll(
+          selectors.facetValue
+        );
+        expect(facetValueElements.length).toBe(expectedFacetValues.length);
+        expectedFacetValues.forEach((facetValue, index) => {
+          expect(facetValueElements[index].item).toEqual(facetValue);
+        });
+      });
+
+      it('should not display the facet card when the facet values have no results', async () => {
+        facetState = {
+          ...facetState,
+          values: [{...exampleFacetValues[0], numberOfResults: 0}],
+        };
+        const element = createTestComponent();
+        await flushPromises();
+
+        const cardContainer = element.shadowRoot.querySelector(
+          selectors.cardContainer
+        );
+        expect(cardContainer).toBeNull();
+      });
```

Comment:
I think this should be in its own `describe('when the facet values have no results')` => `it('should not display the facet card')`